src/components/CalendarMonthGrid.jsx

```diff
@@ -253,8 +262,8 @@ class CalendarMonthGrid extends React.Component {
           isVertical && styles.CalendarMonthGrid__vertical,
           isVerticalScrollable && styles.CalendarMonthGrid__vertical_scrollable,
           isAnimating && styles.CalendarMonthGrid__animating,
-          isAnimating && {
-            transition: 'transform 0.2s ease-in-out',
+          isAnimating && transitionDuration && {
+            transition: `transform ${transitionDuration}s ease-in-out`,
```

Comment:
What do you think about using `ms` here instead of `s` and then making transitionDuration an integer? Also, should the unit be part of the prop name for ultimate clarity? `transitionDurationMS` or something like that?

Reply:
Also, are folks going to want to control the easing? I often find ease-out to be nicer than ease-in-out for example.

Reply:
I'll make the change to ms. I am adding the change to transition duration because it's been requested but it might make sense to control the easing as well. We can do that in a follow-up PR.

Reply:
For sure, but it might be worth considering what the ideal API for this would be before merging in this PR. e.g. do you want it to be 2 separate props, 1 prop that takes an options object, or a string prop that controls all of the transition options at the same time? I think probably 2 separate props, but you would probably know what is better for this project than I.

Reply:
That is fair. I think it would be clearest and make the most sense to add a `transitionEasing` prop that would accept valid options as per the specification. It seems like it would be more confusing to have one prop cover both options.